plugins/sentinelone/komand_sentinelone/util/api.py

```diff
@@ -67,6 +67,11 @@ def apps_by_agent_ids(self, identifiers: str) -> list:
     def agents_action(self, action: str, agents_filter: str) -> dict:
         return self._call_api("POST", AGENTS_ACTION_ENDPOINT.format(action=action), json={"filter": agents_filter})
 
+    def agents_action_move_agent_to_new_site(self, agents_filter: dict, data: dict) -> dict:
+        return self._call_api(
+            "POST", AGENTS_ACTION_ENDPOINT.format(action="move-to-site"), json={"filter": agents_filter, "data": data}
```

Comment:
Would be able to follow the same format as the others here please? 😄 
```suggestion
            "POST", AGENTS_ACTION_ENDPOINT.format(action=action), json={"filter": agents_filter, "data": data}
```